br/pkg/restore/client.go

```diff
@@ -2383,15 +2387,69 @@ func (rc *Client) initSchemasMap(
 	return backupMeta.GetDbMaps(), nil
 }
 
+func initFullBackupTables(
```

Comment:
move the function from `task.initFullBackupTables` to here. The backupschemas is a large file, and needs to be load only when there is no `pitr-id-maps`.